src/main/java/com/kodilla/ecommercee/repositories/UserRepository.java

```diff
@@ -2,6 +2,18 @@
 
 import com.kodilla.ecommercee.domains.User;
 import org.springframework.data.repository.CrudRepository;
+import java.util.List;
+import java.util.Optional;
 
 public interface UserRepository extends CrudRepository<User, Long> {
+    @Override
+    User save (User user);
+
+    @Override
+    Optional<User> findById (Long id);
+
+    Optional<User> findUserByUsername (String username);
+
+    @Override
+    void deleteById (Long id);
```

Comment:
Zbędne jest jawne nadpisywanie metod, bez nadpisywania także są dostępne. dodałbym @Repository oraz @Transactional.

Reply:
Do zostawienia, nie powoduje do konfliktu, a nadpisywanie bylo zrobione zgodnie z przykladami z wczesniejszych modulow.

Reply:
Niepotrzebnie komplikuje kod. We wcześniejszych modułach było napisane dlaczego tam jest to dodawane.

Reply:
Ok, done :)